Meta/CMake/cmake-version.cmake

```diff
@@ -1,6 +1,6 @@
 # Note: Update this alongside Toolchain/BuildCMake.sh
 set(version_ok 0)
-if (CMAKE_VERSION VERSION_GREATER_EQUAL 3.25.1)
+if (CMAKE_VERSION VERSION_GREATER_EQUAL 3.26.4)
```

Comment:
As a side note, do we have to force users to rebuild CMake? As long as there aren't any new required patches that have landed upstream we probably shouldn't.

Reply:
Yeah, I was considering this too, but the comment in that file said to update here at same time, so I just followed it 🤷

But yeah, you're right, no point. I'll update the comment to along those lines